test/SpecializeImageTypes/unused.cl

```diff
@@ -0,0 +1,6 @@
+// RUN: clspv %s -o %t.spv --cl-std=CL2.0 -inline-entry-points
```

Comment:
You need to add `-cl-kernel-arg-info` for the resource var to be produced. The issue didn't manifest itself without this.